src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1009 @@
+			flyoutTVI.IsSelected = false;
+		}
+
+		private void SetupDragToDropTarget_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.Height = 200;
+			TestTreeView2.Visibility = Visibility.Collapsed;
+
+			ContentModeTestTreeView.Height = 200;
+			ContentModeTestTreeView2.Visibility = Visibility.Collapsed;
+
+			TestTreeView.DragItemsStarting += TestTreeView_DragItemsStarting;
+			TestTreeView.DragOver += TreeView_DragOver;
+			TestTreeView.Drop += TreeView_Drop;
+
+			ContentModeTestTreeView.DragItemsStarting += TestTreeView_DragItemsStarting;
+			ContentModeTestTreeView.DragOver += TreeView_DragOver;
+			ContentModeTestTreeView.Drop += TreeView_Drop;
+
+			DraggableElement.DragStarting += Draggable_DragStarting;
+		}
+
+		private void SetupCustomDragUIOverride_Click(object sender, RoutedEventArgs e)
+		{
+			List<TreeViewItem> items = FindVisualChildrenByType<TreeViewItem>(this.TestTreeView);
+			TreeViewItem lastItem = items.Last();
+
+			lastItem.DragEnter += Item_DragEnter;
+			AutomationProperties.SetName(lastItem, "CustomDragUIOverrideDropTarget");
+		}
+
+		private void Item_DragEnter(object sender, DragEventArgs e)
+		{
+			e.DragUIOverride.Caption = "test caption";
+			e.DragUIOverride.IsCaptionVisible = true;
+			e.DragUIOverride.IsGlyphVisible = false;
+			e.DragUIOverride.IsContentVisible = false;
+			e.Handled = true;
+		}
+
+		private async void TreeView_Drop(object sender, DragEventArgs e)
+		{
+			try
+			{
+				var text = await e.DataView.GetTextAsync();
+				Results.Text = "Dropped: " + text;
+			}
+			catch (Exception ex)
+			{
+				ExceptionMessage.Text = ex.ToString();
+			}
+		}
+
+		private void Draggable_DragStarting(UIElement sender, DragStartingEventArgs args)
+		{
+			args.Data.SetText("Test Item");
+			args.Data.RequestedOperation = DataPackageOperation.Copy;
+			Results.Text = "Drag started";
+		}
+
+		private void TreeView_DragOver(object sender, DragEventArgs e)
+		{
+			e.AcceptedOperation = DataPackageOperation.Copy;
+		}
+
+		private void SizeTreeViewsForDrags_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.Height = 200;
+			TestTreeView2.Height = 200;
+		}
+
+		private void DropTarget_DragOver(object sender, DragEventArgs e)
+		{
+			e.AcceptedOperation = DataPackageOperation.Copy;
+		}
+
+		private async void DropTarget_Drop(object sender, DragEventArgs e)
+		{
+			try
+			{
+				var text = await e.DataView.GetTextAsync();
+				DropTargetTextBlock.Text = text;
+			}
+			catch (Exception ex)
+			{
+				ExceptionMessage.Text = ex.ToString();
+			}
+		}
+
+		private void TestTreeView_DragItemsStarting(TreeView sender, TreeViewDragItemsStartingEventArgs e)
+		{
+			var items = new StringBuilder();
+			if (IsInContentMode())
+			{
+				foreach (TreeViewItemSource item in e.Items)
+				{
+					if (items.Length > 0) items.AppendLine();
+					items.Append(item.Content);
+				}
+			}
+			else
+			{
+				foreach (TreeViewNode node in e.Items)
+				{
+					if (items.Length > 0) items.AppendLine();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)